src/constants/index.ts

```diff
@@ -0,0 +1,11 @@
+import { SUPPORTED_MAX_FILE_SIZE } from "src/config";
+
+export const BUCKET_DIRECTORY = "file_bucket";
+
+export enum UPLOAD_TYPE {
+  FAILED = "failed",
+  UPLOADING = "uploading",
+  COMPLETED = "completed",
+}
+
+export const BIG_FILE_SIZE_ERROR_MESSAGE = `file size was exceeded more than ${SUPPORTED_MAX_FILE_SIZE}MBs`;
```

Comment:
For sure we can change it into files. I am thinking as the project goes on, it would be good to have separate constants files for different reasons like messages, for errors, in that way I have created a folder instead of a file.

Reply:
commit - [632024f](https://github.com/parthc2fo/upload-download-thing/pull/1/commits/632024f163625a6b7c6d34b44053e5b6e69cf64a)